gitlab/datadog_checks/gitlab/gitlab.py

```diff
@@ -67,6 +63,31 @@ def check(self, instance):
         for check_type in self.ALLOWED_SERVICE_CHECKS:
             self._check_health_endpoint(instance, check_type, custom_tags)
 
+    def _create_gitlab_prometheus_instance(self, instance, init_config):
+        """
+        Set up the gitlab instance so it can be used in OpenMetricsBaseCheck
+        """
+        # Mapping from Prometheus metrics names to Datadog ones
+        # For now it's a 1:1 mapping
+        # TODO: mark some metrics as rate
```

Comment:
Let do something about this TODO (or let's remove it)

Reply:
I think I'll remove it, the comment already there previously not sure why, but changing types on some metrics would be BC break anyway

Reply:
I think I'll remove it, the comment already there previously not sure why, but changing types on some metrics would be BC break anyway